src/main/java/hudson/plugins/copyartifact/CopyArtifact.java

```diff
@@ -124,7 +126,14 @@ public CopyArtifact(String projectName, String parameters, BuildSelector selecto
 
         this.project = projectName;
         this.parameters = Util.fixEmptyAndTrim(parameters);
-        this.selector = selector;
+
+        if (selector == null) {
+            // TODO: specify the BuildSelector instance in the workflow script via CoreStep?
+            this.selector = new StatusBuildSelector(true);
```

Comment:
Use the `ScriptGenerator` to see. Something like

    selector: [$class: 'StatusBuildSelector', something: true]